bin/packages.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 
 # Get the latest packages data
-rm -f ./data/packages.min.json
+rm -f ./data/packages.min.json ||:
```

Comment:
```suggestion
rm -f ./data/packages.min.json || exit 1
```

is that the same? if so could we be explict about the exit

Reply:
`||:` is shorthand for `|| true`, so if the `rm` call fails, it'll continue siltently.

Reply:
I updated my suggestion, could we use that instead? to be more explict